broadinstitute/pearl
# Fixing ERD relationship

Turns out I had multiple relationships wrong -- this ERD is complicated!  

TO TEST:
1. reset your postgres DB (lots of tables changed)
2. run PopulateCliApp with args `portal` and `portals/ourhealth/portal.json`
3. confirm populate succeeds
